buildSrc/src/main/groovy/com/thoughtworks/go/build/WindowsPackagingTask.groovy

```diff
@@ -30,14 +30,12 @@ class WindowsPackagingTask extends DefaultTask {
   String version
   @Input
   String distVersion
-  @Input
-  Boolean is32Bit = false
 
   Closure<Task> beforePackage
 
   @OutputFile
   public File getOutputFile() {
-    project.file("${project.convention.plugins.get("base").distsDir}/win/${packageName}-${version}-${distVersion}-jre-${flavour()}-setup.exe")
+    project.file("${project.convention.plugins.get("base").distsDir}/win/${packageName}-${version}-${distVersion}-jdk-${flavour()}-setup.exe")
```

Comment:
Lets not change the filenames, a lot of users use scripts that rely on the filename pattern for installing.